Update README.md (#275)
Starcounter/Docs: docs/sql/README.md

# SQL Queries in Starcounter

## Data Manipulation Language (DML) Queries

> Starcounter 3.0 has inherited SQL Query Processor from Starcounter 2.3.2. Please refere to the (original documentation)[https://docs.starcounter.io/v/2.3.2/guides/sql] for the full specification.

Reviewer: ```suggestion
Starcounter 3.0 has the same SQL Query Processor as Starcounter 2.3.2. Please refer to the (original documentation)[https://docs.starcounter.io/v/2.3.2/guides/sql] for the full specification.
```

## Data Definition Language (DDL) Queries

As of right now the following DDL statements are supported:

### Create / drop database index

```sql
CREATE INDEX IX_IndexName ON TableName (ColumnName)
```

```sql
DROP INDEX IX_IndexName ON TableName
```

### Create / drop database table

```sql
CREATE TABLE TableName
(

>     BooleanColumn boolean,

Reviewer: ```suggestion
	BooleanColumn boolean,
```

Author: I did the other way around, changed the rest of the tabs to spaces.

	DecimalColumn decimal,
	DoubleColumn double,
	FloatColumn, float,
	IntColumn int,
	UIntColumn uint,
	TextColumn text
)
```

```sql
DROP TABLE TableName
```

#### Starcounter database vs .NET CLR data types

> Starcounter database data types do not have one to one match with .NET CLR data types.

Reviewer: ```suggestion
There is no one to one match between Starcounter database data types and .NET CLR data types.
```
